inference-engine/scripts/tbb_retro_check.sh

```diff
@@ -0,0 +1,95 @@
+echo "Check OpenVINO binary with different version of TBB"
```

Comment:
how this script is used? why do we need it? At least we need to write some header with such information

Reply:
It's just a helper script to download all TBB versions for last 3 years and check build with that. Has no affect on project build process. I've decided to remove it from patch.